src/components/payment/staff-payment.vue

```diff
@@ -0,0 +1,103 @@
+<template>
```

Comment:
This is what I call an "interface component" that sits between the tab item and the shared component. It exists so that the tab item doesn't need to know about the low level details, and the shared component doesn't need to know about the high level getters/setters/etc. This interface component will be different in different apps.